benchmarks/instruction_counts/definitions/standard.py

```diff
@@ -97,14 +159,57 @@
         x[True] = y[True]                        | x.index_put_({true}, y.index({true}));
     """)),
 
+    "Metadata and views": GroupedVariants(*parse_stmts(r"""
+        Python                                   | C++
+        ---------------------------------------- | ----------------------------------------
+        # @setup                                 | // @setup
+        x = torch.ones((4, 4))                   | auto x = torch::ones({4, 4});
+                                                 |
+        # @size                                  | // @size
+        x.size()[0]                              | x.sizes()[0];
+                                                 |
+        # @stride                                | // @stride
+        x.stride(0)                              | x.stride(0);
+                                                 |
+        # @as_strided                            | // @as_strided
+        torch.as_strided(x, (2, 3), (4, 1), 2)   | torch::as_strided(x, {2, 3}, {4, 1}, 2);
+                                                 |
+        # @select                                | // @select
+        x.select(1, 1)                           | x.select(1, 1);
+                                                 |
+        # @unsqueeze                             | // @unsqueeze
+        x.unsqueeze(0)                           | x.unsqueeze(0);
+                                                 |
+        # @view                                  | // @view
+        x.view(-1, 1)                            | x.view({-1, 1});
```

Comment:
can you please add `.reshape` here? It's much more expensive than `view` now, would be good to track and fix it

Reply:
Done. Also, thanks for filing https://github.com/pytorch/pytorch/issues/55126